### ISSUE TITLE
use black for formatting

### ISSUE THREAD

**Issue by nerdoc** [IlyaSemenov/django-modelsettings#17]
when more than one people are working on one project, it's not easy to maintain code "similarity"... Black solves this issue. While it's not perfect for everyone and every case, it's an easy way to keep a common layout.

I don't want to provide a PR here, as it touches MANY lines at once, and it would look like I had written those lines in it history.

So if you are ok with this, just install black, list it as dev requirement and run a `black . ` in the root dir...?

**Comment by IlyaSemenov** [comment]
Yes, I'm okay with black. I ended up using it in my repos. Even though I'm a strong believer in tabs, I learned to accept black due to lack of a reasonable alternative.

I'll reformat the files.
